Dfe.Academies.External.Web/Services/ConversionApplicationCreationService.cs

```diff
@@ -38,17 +38,32 @@ public async Task AddSchoolToApplication(int applicationId, int schoolUkUrn, str
     {
 	    try
 	    {
-		    //ConversionApplicationApiPostResult result;
             ResilientRequestProvider apiRequestProvider = new (_httpClientFactory.CreateClient(AcademisationAPIHttpClientName));
 		    SchoolApplyingToConvert school = new(name, schoolUkUrn, applicationId,null);
 
-			// TODO: await API response from Academisation API
-			//result = await apiRequestProvider.PostAsync<ConversionApplicationApiPostResult, SchoolApplyingToConvert>(apiurl, school);
-	    }
-	    catch (Exception ex)
+            // TODO: wire up Academisation API
+            // var result = await apiRequestProvider.PostAsync<ConversionApplicationApiPostResult, SchoolApplyingToConvert>(apiurl, school);
```

Comment:
I think this is probably going to be a put of the whole application - can change later though.